apps/client/package.json

```diff
@@ -47,11 +47,7 @@
     "tsconfig": "*"
   },
   "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
+    "production": [">0.2%", "not dead", "not op_mini all"],
```

Comment:
I noticed this one keeps changing between single and multiple lines. Do you happen to know what causes it?

Reply:
Not sure, but it seems like one of us has local overrides? 🤔